04/index.js

```diff
@@ -1,29 +1,24 @@
-
-const ul = document.getElementById('js-lists');
+const ul = document.getElementById("js-lists");
 const fragment = document.createDocumentFragment();
 
-const contents = [
-  { href: 'a1.html', src: '/img/bookmark.png', text: 'a1' },
-  { href: 'a2.html', src: '/img/message.png', text: 'a2' }
+const contents =   [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" },
 ];
 
-for(let i = 0; i < contents.length; i++){
-  const image = document.createElement('img');
-  const anchor = document.createElement('a');
-  const li = document.createElement('li');
-
-  const content = contents[i];
-
-  image.src = content.src;
-  anchor.href = content.href;
-
-  li
-  .appendChild(anchor)
-  .appendChild(image);
+contents.forEach ((content)=> {
```

Comment:
You can write like this.
contents.forEach (content => {

Reply:
I learned that parentheses are not necessary when there is one argument.
a009234be66cbdadefaa52831e70673b9c0eb585